README.rst

```diff
@@ -39,7 +39,10 @@ Table of Contents:
 Quickstart
 ----------
 
-Cachalot officially supports Python 3.5-3.9 and Django 2.0-2.2, 3.0-3.1 with the databases PostgreSQL, SQLite, and MySQL.
+Cachalot officially supports Python 3.6-3.9 and Django 2.2, 3.0, 3.1 and 3.2 with the databases PostgreSQL, SQLite, and MySQL.
+
+Newer versions of Django 3.x should work as well, but due to cachalot monkey-patching the Django ORM, using a version not
+officially tested and supported might cause unexpected breakage.
```

Comment:
```suggestion
No upper limit is imposed by cachalot. When Django's async ORM or cache is rolled out, there may be huge changes and breakages due to cachalot monkey-patching.
```

With huge monkey patching, I'd be cautious about this. I've had instances of subquery just failing after a Django version minor update. This is either due to the test cases failing to catch certain edge cases, the test suite being incomplete, or usually the ORM is changing somehow.

Additionally, Django 3.3 or 3.4 will introduce the async ORM or the async cache (most likely the cache will come in Django 3.3, but for some reason my PR is within the Django 3.2 updates... oh well) which means monkey patching is going to cause hell for cachalot. Luckily, Andrew hasn't started the async portion as of yet, but the roadmap is saying it'll happen pretty soon.